src/PageContent/Calendar/Header.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { DeliveryType } from '../../Enums';
+import './Header.css';
+import white_cross from '../../icons/white_cross.svg';
+
+class Header extends Component {
+    render() {
+        let deliveryType = '';
+        if (this.props.eventType === DeliveryType.RECURRING) {
+            deliveryType = 'Recurring Delivery';
```

Comment:
you'll also want to pass down either `account` or `accountType` (`account.accountType`) from `PageContainer` so that you can:

- change wording of the title b/w "Pick-up" vs "Delivery"
- decide what to show in the content section

Remember to compare it to the `AccountType` enum